examples/db-templates/redis-ephemeral-template.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Password: ${REDIS_PASSWORD}\n Connection URL: redis://${DATABASE_SERVICE_NAME}:6379/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/redis-container/blob/master/3.2.",
   "labels": {
-    "template": "redis-ephemeral-template"
+    "template": "redis-ephemeral-template",
+    "app": "redis-ephemeral-template"
```

Comment:
The app label should be parameterized so that when a DB template is used with another application and we want to associate it with that app we can.